engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -2073,6 +2090,8 @@ static uintptr_t GetExtProcAddress(const char* name, const char* extension_name,
                 // attach the texture to FBO color attachment point
                 glFramebufferTexture2D(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0 + i, GL_TEXTURE_2D, rt->m_ColorBufferTexture[i]->m_Texture, 0);
                 CHECK_GL_ERROR;
+
+                // printf("Creating RT at %d with %d \n", GL_COLOR_ATTACHMENT0 + i, rt->m_ColorBufferTexture[i]->m_Texture);
```

Comment:
Might be worth having a "DEBUG" define at the top of the file, for easy enabling?